### PR TITLE
Patch

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,12 +15,17 @@ If you don't have YunoHost, please consult [the guide](https://yunohost.org/#/in
 
 ## Overview
 
-Terminal in browser over HTTP/HTTPS. (Ajaxterm/Anyterm alternative)
+Terminal over HTTP and HTTPS. WeTTy is an alternative to ajaxterm and anyterm but much better than them because WeTTy uses xterm.js which is a full fledged implementation of terminal emulation written entirely in JavaScript. WeTTy uses websockets rather then Ajax and hence better response time.
```
Comment (Salamandar) [inline README.md:18]: Why this change ?

File: scripts/restore [hunk 2]
```diff
@@ -36,8 +36,6 @@ final_path=$(ynh_app_setting_get --app=$app --key=final_path)
 #=================================================
 ynh_script_progression --message="Validating restoration parameters..." --weight=1
 
-ynh_webpath_available --domain=$domain --path_url=$path_url \
-	|| ynh_die --message="Path not available: ${domain}${path_url}"
```
Comment (Salamandar) [inline scripts/restore:40]: Why remove that ? This looks kinda necessary.

File: README.md (hunk 1 above)
Comment (Salamandar) [reply README.md:18]: I mean the current "overview" is already enough and this just adds "noise".

File: scripts/restore (hunk 2 above)
Comment (ericgaspar) [reply scripts/restore:40]: https://github.com/YunoHost/example_ynh/pull/171

File: README.md (hunk 1 above)
Comment (ericgaspar) [reply README.md:18]: `Why this change ?` 
why not... :) We now have a Description.md available for more in depth description (instead of using the description from the manifest.)

File: scripts/restore (hunk 2 above)
Comment (Salamandar) [reply scripts/restore:40]: ok !
